demos/jans-tent/.flaskenv

```diff
@@ -0,0 +1,2 @@
+#.flaskenv
+FLASK_APP=clientapp
```

Comment:
did you mean to commit this file ?

Reply:
Yes, it's to be used with `flask` or `python -m flask` command, when stable "production" is needed.